service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/RespondentSolicitorContentProvider.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
+import uk.gov.hmcts.reform.fpl.model.notify.submittedcase.RespondentSolicitorTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.SharedNotifyContentProvider;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class RespondentSolicitorContentProvider extends SharedNotifyContentProvider {
+
+    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookup;
+
+    public RespondentSolicitorTemplate buildRespondentSolicitorSubmissionNotification(
+        CaseData caseData, RespondentSolicitor representative) {
+
+        return RespondentSolicitorTemplate.builder()
+            .representativeName(representative.getFirstName() + " " + representative.getLastName())
```

Comment:
in #2476 this looks like:
`.representativeName(representative.getLastName())`

which is correct? if you want first name too, we might need some validation to make it mandatory (see that soorya added validation for last name)
see: https://github.com/hmcts/fpl-ccd-configuration/pull/2476/files#r610480339

Reply:
@joez-f business confirmed that if the respondent’s name isn’t filled in, there’s no “Dear xxx” line – just straight into the body of the email.
I've updated the email notification template and updated the code according to this change.